packages/dx-react-scheduler/docs/reference/scheduler.md

```diff
@@ -0,0 +1,57 @@
+# Scheduler Reference
+
+The Scheduler is a root container component designed to process and display data specified via the `data` property. The Scheduler's functionality (data visualization and data processing) is implemented in several plugins specified as child components.
+
+## Import
+
+Use the following statement to import a plugin with embedded theme components:
+
+```js
+import { Scheduler } from '@devexpress/dx-react-scheduler-material-ui';
+```
+
+If you want to use custom components, you can import the themeless plugin:
+
+```js
+import { Scheduler } from '@devexpress/dx-react-scheduler';
+```
+
+## User reference
+
+### Properties
+
+Name | Type | Default | Description
+-----|------|---------|------------
+data | Array&lt;object&gt; | | An array containing custom data. A user defines the access to this data.
```

Comment:
Why `custom data`?

Reply:
@SergeyAlexeev I changed the AppointmentModel Interface.

I suggest to define additonal properties by this [example](https://www.typescriptlang.org/docs/handbook/interfaces.html) 

```ts
interface SquareConfig {
    color?: string;
    width?: number;
    [propName: string]: any; // Additional properties
}
```